frontend/test/__support__/e2e/cypress.js

```diff
@@ -1,9 +1,35 @@
 require("cypress-grep")();
 
+import addContext from "mochawesome/addContext";
 import "@testing-library/cypress/add-commands";
 import "cypress-real-events/support";
 import "@cypress/skip-test/support";
 import "@percy/cypress";
 import "./commands";
 
 Cypress.on("uncaught:exception", (err, runnable) => false);
+
+Cypress.on("test:after:run", (test, runnable) => {
+  if (test.state === "failed") {
+    const titleToFileName = title => title.replace(/[>]/g, "");
```

Comment:
Correct me if I'm wrong, but the way I understand this - it should be a function to sanitize strings that will be used as the file name? Naming of the function could ideally suggest that.

Also, we're currently using `foo > bar` in `describe` blocks but there are other characters that are illegal in a file name. When/if the need arises, we can expand the function to filter them out as well (or we can find a completely different mechanism to generate the file name).